training/nvidia-bootc/Containerfile

```diff
@@ -157,7 +157,11 @@ fi
 RUN grep -q /usr/lib/containers/storage /etc/containers/storage.conf || \
     sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
 	/etc/containers/storage.conf && \
-	cp /run/.input/ilab /usr/local/bin/ilab
+	if [ -f "/run/.input/ilab" ]; then \
+	  cp /run/.input/ilab /usr/local/bin/ilab \
+        else \
+	  curl -o /usr/local/bin/ilab "https://raw.githubusercontent.com/containers/ai-lab-recipes-upstream/main/training/ilab-wrapper/ilab" \
```

Comment:
Is containers/ai-lab-recipes-upstream a new repo?

Reply:
Excellent catch, it is not.